.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "6"
+script: npm run postinstall
+script: node index.js
```

Comment:
Per default it runs `npm install` and then `npm test`but if you dont have any test you might configure something like I did.